searchlib/src/vespa/searchlib/attribute/reference_attribute.cpp

```diff
@@ -336,24 +342,42 @@ ReferenceAttribute::notifyReferencedRemove(const GlobalId &gid)
     EntryRef ref = _store.find(gid);
     if (ref.valid()) {
         const auto &entry = _store.get(ref);
+        uint32_t oldReferencedLid = entry.lid();
         _referenceMappings.notifyReferencedRemove(entry);
+        if (oldReferencedLid != 0) {
```

Comment:
Is this case tested in reference_attribute_test.cpp? If not, consider adding new test case.